karate-core/src/main/java/com/intuit/karate/core/ScenarioResult.java

```diff
@@ -106,6 +107,16 @@ private static void recurse(List<Map> list, StepResult stepResult, int depth) {
                 call.setLine(stepResult.getStep().getLine());
                 call.setPrefix(StringUtils.repeat('>', depth));
                 call.setText(fr.getCallNameForReport());
+                for(ScenarioResult scenarioResult: fr.getScenarioResults()){
```

Comment:
instead of all this - just don't invoke this line. problem solved

`call.setDocString(fr.getCallArgPretty());`

Reply:
if that works, just make that change, we don't need any tests

Reply:
Yeah I though of that but looking at original poster's issue, I masked the value just in case if someone wants to ensure that config values are read by the code but not visible in the report.
Ex: I want see that 4 fields are defined and read by the code. So they get printed but value is hidden.
key 1 = ***
Key 2 = *** and so on. Let me know if that is not needed then I will just ignore calling the 
```
call.setDocString(fr.getCallArgPretty());
```

Reply:
yes my preference is to keep this one simple. in the long term we can do fine grained control of things